rmf_human_detector/launch/human_detector_launch.py

```diff
@@ -0,0 +1,73 @@
+import launch
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch_ros.actions import Node
+from launch.substitutions import EnvironmentVariable
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from ament_index_python.packages import get_package_share_directory
+
+import os
+
+def generate_launch_description():
+    # Bridge
+    bridge = Node(
```

Comment:
Understand this bridge is useful for testing in simulation but in general it is specific to the setup of the sim and hence should only be in a launch file that launches a simulation world. Please remove it from here. You should add it to hotel world launch file.